docs/spl-docs/head-command.md

```diff
@@ -0,0 +1,105 @@
+# head command
+
+## Description
+
+This command retrieves the initial N results as they appear in the search sequence. Two forms of constraints can be utilized: a fixed count of results or a conditional expression that continues to return results until the specified condition is no longer met.
+
+## Syntax
+The required syntax is in **bold**.
+
+**head**\
+[\<N\> | (\<eval-expression\>)]\
+[limit=\<int\>]\
+[null=\<bool\>]\
+[keeplast=\<bool\>]
+
+
+### Required Arguments
+
+None.
+
+
+### Optional Arguments
+
+#### \<N\>
+**Syntax:** `<int>`\
+**Description:**\
+The number of results to return.\
+**Default:** 10
+
+
+
+#### limit
+**Syntax:** `limit=<int>`\
+**Description:**\
+An alternative method to specify the number of results to be returned.\
+**Default:** 10
+
+
+
+#### \<eval-expression\>
+**Syntax:** `<eval-compare-exp> | <eval-bool-exp>`\
+**Description:** \
+`<eval-expression>` is a boolean expression that evaluates to `true` or `false`.\
+The search continues to yield results until the given expression is no longer `true`.
+
```

Comment:
I think it's valid to give both a numeric limit and an `eval-expression`. If so, it would be good to mention here how `head` behaves in that scenario

Reply:
addressed in the latest commit.